src/vs/workbench/browser/parts/editor/editorPart.ts

```diff
@@ -240,7 +241,18 @@ export class EditorPart extends Part implements IEditorPart, IEditorGroupService
 		// while the UI is not yet ready. Clients have to deal with this fact and we have to make sure that the
 		// stacks model gets updated if any of the UI updating fails with an error.
 		const group = this.ensureGroup(position, !options || !options.preserveFocus);
-		const pinned = !this.previewEditors || (options && (options.pinned || typeof options.index === 'number')) || input.isDirty();
+		let pinned = !this.previewEditors || (options && (options.pinned || typeof options.index === 'number')) || input.isDirty();
+		if (this.previewEditors) {
+			pinned = !this.configurationService.getConfiguration<IWorkbenchEditorConfiguration>().workbench.editor.enablePreviewFromQuickOpen;
```

Comment:
> @wprater you can use most service from most classes by just adding it to the constructor as I was showing it for quickOpenController. We use constructor service injection and you typically do not have to deal with it.

good to know.  in this case, I have to explicitly pass it down to the `EditorQuickOpenEntry` during the super call for all the abstract classes that extend it.  not really a good approach. 

I would rather put all this stuff in the abstract classes `getOptions`.